### PR TITLE
Issue 1 create an entry. Edwin Tan

### PR DESCRIPTION
**Pull request by edtan094**
Code-Journal-issue-1-create-an-entry-exercise

### CODE REVIEW

**Comment by edtan094** [comment]
Code-Journal-issue-1-create-an-entry-exercise 2

**Comment by edtan094** [comment]
Code-Journal-issue-1-create-an-entry-exercise 3

**Comment by edtan094** [comment]
Code-Journal-issue-1-create-an-entry-exercise 4

File: index.html [hunk 1]
```diff
@@ -5,19 +5,56 @@
...
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link href="https://fonts.googleapis.com/css2?family=Proza+Libre&display=swap" rel="stylesheet">
 </head>
-<body>
-  <header>
+<body class="gray-background">
+  <header class="purple-background">
+    <div class="container row">
```
Comment (codingcodymiller) [inline index.html]: `container` and `row` on one element?

File: index.html [hunk 2]
```diff
@@ -5,19 +5,56 @@
...
+      <div class="column-full">
+        <h4 class="white-text">Code Journal</h4>
+      </div>
+    </div>
   </header>
   <main>
+    <div class="container" data-view="entry-form">
+      <div class="row column-full">
```
Comment (codingcodymiller) [inline index.html]: `row` and `column-full` on one element?

File: js/data.js [hunk 3]
```diff
@@ -6,3 +6,17 @@ var data = {
...
+var previousEntriesJSON = localStorage.getItem('entry');
+var previousNextEntryId = localStorage.getItem('entry ID');
+if (previousEntriesJSON !== null && previousNextEntryId !== 1) {
+  data.entries = JSON.parse(previousEntriesJSON);
+  data.nextEntryId = JSON.parse(previousNextEntryId);
+}
+
+window.addEventListener('beforeunload', function () {
+  var savedImagesJSON = JSON.stringify(data.entries);
+  var savedNextEntryId = JSON.stringify(data.nextEntryId);
```
Comment (codingcodymiller) [inline js/data.js]: You should just save the entire `data` object, not these separate properties.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**